modules/stic_Events/Utils.php

```diff
@@ -318,12 +318,15 @@ public static function createPeriodicSessions()
             if (isset($_REQUEST['color']) && $_REQUEST['color'] != '') {
                 $sessionBean->color = $_REQUEST['color'];
             } else {
-                $sessionBean->color = $user;
+                $sessionBean->color;
             }
             if (isset($_REQUEST['description']) && $_REQUEST['description'] != '') {
                 $sessionBean->description = $_REQUEST['description'];
-            } else {
-                $sessionBean->description = $user;
+            }
+             else {
+                $eventBean = BeanFactory::getBean('stic_Events', $sessionBean->stic_sessions_stic_eventsstic_events_ida);
```

Comment:
Supongo que lo habéis hablado con Alberto. Me ha sorprendido que se copie la descripción del evento, yo la hubiera dejado en blanco, que de hecho me parece que es lo que se hacía antes de incorporar el tema color y la descripción en el asistente.

Reply:
Corregido, se deja en blanco la descripción